src/components/GenericButton.tsx

```diff
@@ -10,6 +10,8 @@ type SharedButtonProps = {
   square?: boolean;
   onClick?: () => void;
   className?: string;
+  highlightIcon?: boolean;
```

Comment:
What's this do?

Reply:
That's a residue from a previous concept. Removed that.